src/Surfnet/ServiceProviderDashboard/Domain/Entity/Entity/Logo.php

```diff
@@ -68,4 +69,13 @@ public function merge(Logo $logo): void
         $this->width = is_null($logo->getWidth()) ? null : $logo->getWidth();
         $this->height = is_null($logo->getHeight()) ? null : $logo->getHeight();
     }
+
+    public function asArray(): array
+    {
+        return [
+//            'metaDataFields.logo:0:height' => $this->getHeight(),
+//            'metaDataFields.logo:0:width' => $this->getWidth(),
```

Comment:
Commented code is not allowed :stuck_out_tongue: 
Or maybe add a comment to clarify why?

Reply:
`rm -rf /`